source/guides/ember-cli/glossary.md

```diff
@@ -0,0 +1,84 @@
+Joining a web development community can be a challenge within itself, especially when all the resources you visit assume you're familiar with other technologies that you're not familiar with.
+
+Our goal is to help you avoid that mess and come up to speed as fast as possible; you can consider us your internet friend.
+
+##CDN
+Content Delivery Network
```

Comment:
BTW, since Markdown is a superset of HTML, you can use `<table>` etc. tags here and it will "just work." I make no guarantees about the CSS styling. ;)